cumulus_library/actions/exporter.py

```diff
@@ -56,13 +70,28 @@ def export_study(
         description=f"Exporting {manifest_parser.get_study_prefix()} data...",
     ):
         query = f"SELECT * FROM {table}"
-        dataframe = db.execute_as_pandas(query)
+        # Note: we assume that, for duckdb, you are unlikely to be dealing with large
+        # exports, so it will ignore the chunksize parameter, as it does not provide
+        # a pandas enabled cursor.
```

Comment:
This feels like a comment that might more naturally fit in the duckdb database implementation of `execute_as_pandas`.

Reply:
i think it's in both places - but i could remove it from here.